python-sdk/src/forta_agent/utils.py

```diff
@@ -81,6 +82,24 @@ def create_transaction_event(dict):
     return TransactionEvent(dict)
 
 
+def get_alerts(dict):
+    from .forta_graphql import AlertQueryOptions
+    forta_api = "https://api.forta.network/graphql"
+    headers = {"content-type": "application/json"}
+    query_options = AlertQueryOptions(dict)
+    payload = query_options.get_query()
+    response = requests.request("POST", forta_api, json=payload, headers=headers)
+
+    if response.status_code == 200:
+        data = response.json().get('data')
+
+        if data:
+            return data.get('alerts')
```

Comment:
will the paging info also be returned? (or just alerts?)

Reply:
Yep the page info is returned. The returned object looks like this:

`{
 alerts: [...],
 pageInfo: {..}
}`

In the image in the PR description you can see what the pageInfo looks like in the response.